.travis.yml

```diff
@@ -1,5 +1,11 @@
 language: objective-c
 
+sudo: false
```

Comment:
@SpainTrain All of the projects @steelbrain _just_ enabled Travis on for me a week or two ago needed that explicitly stated. I'm pretty sure it still isn't the default until they get `sudo` working inside the containers.